CHANGELOG.md

```diff
@@ -65,24 +65,36 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/),
 
 **Added:**
 
-- `String handling` attribute
-- `ChargeClass` column
-- `CommitmentDiscountStatus` column
-- `ContractedCost` column
-- `ContractedUnitPrice` column
-- `RegionId` column
-- `RegionName` column
-- `DataGenerator` metadata property
-- `CreationDate` metadata schema property
-- `FocusVersion` metadata schema property
-- `SchemaId` metadata schema property
-- `ColumnName` metadata column definition property
-- `DataType` metadata column definition property
-- `NumberScale` metadata column definition property
-- `NumberPrecision` metadata column definition property
-- `ProviderTagPrefix` metadata column definition property
-- `StringEncoding` metadata column definition property
-- `StringMaxLength` metadata column definition property
+New use cases:
+- Compare billed cost per subaccount to budget
```

Comment:
Note to self: I need to show how we flag use cases that have been impacted by a change. I don't think this use case has been changed yet, tho.